packages/foreman/pcp-mmvstatsd/pcp-mmvstatsd.spec

```diff
@@ -65,3 +57,5 @@ exit 0
 %{_unitdir}/%{name}.service
 %config(noreplace) %{_sysconfdir}/default/%{name}
 %attr(755, %{name}, %{name})%{_sharedstatedir}/%{name}
+
+%changelog
```

Comment:
I think it was missing this.

Reply:
Yes -- please add a changelog entry.

Reply:
Can you elaborate? Do you want me to add only old entries or all entries? I thought that tito was adding those automatically. I will add this into README once confirmed.